cloud-sql/sql-server/sqlalchemy/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Google, LLC.
```

Comment:
```suggestion
# Copyright 2020 Google, LLC.
```

Reply:
Could you bump the copyright year to 2020  in these files (unless they're being copied from somewhere else)?